cli/module_registry_test.go

```diff
@@ -0,0 +1,49 @@
+package cli
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"go.viam.com/test"
+
+	"go.viam.com/rdk/testutils"
+	"go.viam.com/rdk/testutils/inject"
+)
+
+func TestUpdateModelsAction(t *testing.T) {
+	_, filename, _, ok := runtime.Caller(0)
+	test.That(t, ok, test.ShouldBeTrue)
```

Comment:
[opt] We have a helper for finding absolute paths: https://github.com/viamrobotics/rdk/blob/main/utils/file.go#L17.